__tests__/app.test.js

```diff
@@ -38,25 +38,67 @@ describe("/api/topics", () => {
 });
 
 describe("/api", () => {
-      test("200: responds with an object describing all the available endpoints on the API", () => {
-        return request(app)
-          .get("/api")
-          .expect(200)
-          .then(({ body }) => {
-            const endpointKeys = Object.keys(body);
-            expect(endpointKeys[0]).toBe("GET /api");
-            expect(endpointKeys[1]).toBe("GET /api/topics");
-            expect(endpointKeys[2]).toBe("GET /api/articles");
-            const endpoints = [
-              body["GET /api"],
-              body["GET /api/topics"],
-              body["GET /api/articles"],
-            ];
-            endpoints.forEach((endpoint) => {
-                expect(endpoint.hasOwnProperty('description')).toBe(true)
-            })
-          });
+  test("GET 200: responds with an object describing all the available endpoints on the API", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
+        const endpointKeys = Object.keys(body);
+        expect(endpointKeys[0]).toBe("GET /api");
+        expect(endpointKeys[1]).toBe("GET /api/topics");
+        expect(endpointKeys[2]).toBe("GET /api/articles");
+        const endpoints = [
+          body["GET /api"],
+          body["GET /api/topics"],
+          body["GET /api/articles"],
+        ];
+        endpoints.forEach((endpoint) => {
+          expect(endpoint.hasOwnProperty("description")).toBe(true);
+        });
       });
-   
+  });
+});
 
+describe("/api/articles/:article_id", () => {
+  test("GET: 200 should return a single article object", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        const article = body.article;
+        expect(typeof article).toBe("object");
```

Comment:
If we are going to test that the typeof the article is an object, then we should be checking that it isn't an array